src/makeCalculator.js

```diff
@@ -5,6 +5,33 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+    add(operands) {
+      return (calculator.result += operands);
+    },
```

Comment:
```suggestion
    add(operands) {
      return (this.result += operands);
    },
```
better to use `this` instead of `calculator` it will improve reusability